amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -8,492 +8,967 @@ input AMPLIFY {
-  id: ID # nullable since resolver will use User / Colony id by default
+  """The unique identifier for the user profile."""
+  id: ID
+  """The URL of the user's avatar image."""
   avatar: String
+  """The URL of the user's thumbnail image."""
   thumbnail: String
+  """The display name of the user."""
   displayName: String
+  """A short description or biography of the user."""
   bio: String
+  """The user's location (e.g., city or country)."""
   location: String
+  """The user's personal or professional website."""
   website: AWSURL
+  """The user's email address."""
   email: AWSEmail
+  """Any additional metadata or settings related to the user profile."""
   meta: ProfileMetadataInput
 }
 
+"""
+Input data for a user's reputation within a Domain in a Colony. If no `domainId` is passed, the Root Domain is used.
+A `rootHash` can be provided, to get reputation at a certain point in the past.
+"""
 input GetUserReputationInput {
+  """The Ethereum wallet address of the user."""
   walletAddress: String!
+  """The Ethereum address of the Colony."""
   colonyAddress: String!
+  """The ID of the Domain within the Colony. If not provided, defaults to the Root Domain."""
   domainId: Int
+  """The root hash of the reputation tree at a specific point in time."""
   rootHash: String
 }
 
+"""
+Input data for updating an extension's information within a Colony, based on the Colony ID and extension hash.
+The hash is generated like so: `keccak256(toUtf8Bytes(extensionName))`, where `extensionName` is the name of the extension contract file in the Colony Network
+"""
 input UpdateExtensionByColonyAndHashInput {
+  """The unique identifier for the Colony."""
   colonyId: ID!
+  """The hash of the extension to be updated."""
   hash: String!
+  """A flag to indicate whether the extension is deprecated."""
   isDeprecated: Boolean
+  """A flag to indicate whether the extension is deleted."""
   isDeleted: Boolean
+  """A flag to indicate whether the extension is initialized."""
   isInitialized: Boolean
+  """The version of the extension."""
   version: Int
+  """The Ethereum address of the user who installed the extension."""
   installedBy: String
+  """The timestamp when the extension was installed."""
   installedAt: AWSTimestamp
 }
 
+"""
+Input data to store the latest available version of the core Colony contract and available extensions
+
+The extension hash is generated like so: `keccak256(toUtf8Bytes(extensionName))`, where `extensionName` is the name of the extension contract file in the Colony Network (e.g. `VotingReputation`)
+"""
 input SetCurrentVersionInput {
+  """COLONY for the Colony contract, extension hash for extensions"""
   key: String!
+  """Latest available version"""
   version: Int!
 }
 
+"""
+Return type for tokens gotten from DB or from chain.
+"""
 type TokenFromEverywhereReturn {
+  """List of tokens found"""
   items: [Token]
 }
 
+"""
+Input data for retrieving a user's reputation within the top domains of a Colony.
```

Comment:
I was wondering what "top domains" means here. Is it all domains or only the top X domains?